src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -593,7 +593,25 @@ public synchronized void applyDML(Sequenced<DMLEvent> sequencedEvent) throws Exc
       .setTableName(normalizedTableName)
       .build();
     long sequenceNumber = sequencedEvent.getSequenceNumber();
-    gcsWriter.write(new Sequenced<>(normalizedDMLEvent, sequenceNumber));
+    try {
+      Failsafe.with(new RetryPolicy<>()
```

Comment:
Can we create the retry policy object as a field in BigQueryEventConsumer, maybe call it gcsWriterRetryPolicy ?